2022 - Imperialis Militia.cat

```diff
@@ -10819,22 +10798,21 @@ equal to the number of Militia Gunners in the unit when making a Shooting Attack
               </modifiers>
             </infoLink>
             <infoLink name="Galvanic Traction Drive" hidden="false" type="rule" id="ca9e-6957-cb91-fbf8" targetId="e51e-cae1-d9e7-d317"/>
+            <infoLink name="Krios Battle Tank" hidden="false" type="profile" id="9414-f04e-affd-3bf4" targetId="0d8f-f51e-e76b-e6aa">
```

Comment:
Re the other naming comment, like you've done here. Github should add a "well done" feature because this looks nice here.